lib/client-assets.php

```diff
@@ -685,3 +685,17 @@ function gutenberg_extend_settings_block_patterns( $settings ) {
 	return $settings;
 }
 add_filter( 'block_editor_settings', 'gutenberg_extend_settings_block_patterns', 0 );
+
+/**
+ * Extends block editor settings to determine whether to use custom unit controls.
+ * Currently experimental.
+ *
+ * @param array $settings Default editor settings.
+ *
+ * @return array Filtered editor settings.
+ */
+function gutenberg_extend_settings_custom_units( $settings ) {
+	$settings['__experimentalDisableCustomUnits'] = get_theme_support( 'disable-custom-units' );
```

Comment:
Should this support a second argument where the theme can say which should be the unit shown / used? Like `get_theme_support( 'disable-custom-units', 'rem' )` would force rem to be used in all unit-based selectors.

Reply:
Good suggestion! Let me make that adjustment ✨